### PR TITLE
SB-21611: certificate registry , cert-service api doc

### PR DESCRIPTION
**Pull request by aishwa8141**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://614.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://614.qa.docs.sunbird.org/api-documentation/apis/certificateregistry// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://614.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://614.qa.docs.sunbird.org/api-documentation/apis/certificateregistry// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://614.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificateregistry// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificatetemplateapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://614.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificateregistry// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificatetemplateapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://614.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificateregistry// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificatetemplateapi// <br />  <br/> **Modified sections are -** <br />

File: apis/certificate/certificate.yaml [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-  title: Certificate api
-  description: This api is used for digital certificate assertion.
+  title: Certificate Generate API
+  description: |- 
+        This api is used for digital certificate assertion.
+        - [Testcases](https://app.getpostman.com/run-collection/4d683e7b17b98026c267)
   version: "1.0"
 servers:
-- url: https://staging.open-sunbird.org/api/certs/v1
+- url: https://staging.open-sunbird.org/api/certs
```
Comment (PradyumnaNagendra) [inline apis/certificate/certificate.yaml:9]: let the url be https://staging.open-sunbird.org/api
and add the complete paths after this for all the APIs
For eg., for cert generate, path will be `/cert/v1/certs/generate`

File: apis/certificate/certificate.yaml [hunk 2]
```diff
@@ -15,13 +17,15 @@ paths:
         This API is associated with digitally certificate generation.
            - The **_ /generate_** endpoint executes the "Certificate" request based on parameters provided as metadata in the request body to be replaced by appropriate Id
```
Comment (PradyumnaNagendra) [inline apis/certificate/certificate.yaml:18]: Add the backend route in the description

File: apis/certificateregistry/certificateregistryapi.yaml [hunk 3]
```diff
@@ -1,262 +1,766 @@
...
-      - name: Authorization
-        in: header
-        description: To make use of the API, you require authorization. Raise a request
-          to the administrator for the use of the API. You will receive the authorization
-          key. Specify the key received, here.
-        required: true
-        schema:
-          type: string
+           - The request accepts id, accessCode , jsonUrl, jsonData(the raw certificate json),related (courseId, batchId, type of the certificate, templateId(optional)). The difference between v1 and v2 are v1 expects pdfurl and v2 expects printUri(materialised svg string) in the jsonData
+           - Backend route : /certs/v2/registry/add
```
Comment (PradyumnaNagendra) [inline apis/certificateregistry/certificateregistryapi.yaml:230]: Give the complete url for Backend route

**Review by PradyumnaNagendra — CHANGES_REQUESTED** [review]

File: apis/certificate/certificate.yaml (hunk 1 above)
Comment (aishwa8141) [reply apis/certificate/certificate.yaml:9]: done

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://614.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificateregistry// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificatetemplateapi// <br />  <br/> **Modified sections are -** <br />

File: apis/certificateregistry/certificateregistryapi.yaml (hunk 3 above)
Comment (aishwa8141) [reply apis/certificateregistry/certificateregistryapi.yaml]: done

File: apis/certificate/certificate.yaml (hunk 2 above)
Comment (aishwa8141) [reply apis/certificate/certificate.yaml:18]: done

**Review by PradyumnaNagendra — APPROVED** [review]

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://614.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificateregistry// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificatetemplateapi// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @aishwa8141,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://614.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificate// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificateregistry// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis/certificatetemplateapi// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis// <br /> - http://614.qa.docs.sunbird.org/api-documentation/apis// <br />  <br/> **Modified sections are -** <br />

**Comment by Sunitashelar** [comment]
Merged the PR.

**Pull request merged**